lib/generators/react_on_rails/setup_files_generator.rb

```diff
@@ -0,0 +1,26 @@
+require "rails/generators"
+
+module ReactOnRails
+  module Generators
+    class SetupFilesGenerator < Rails::Generators::Base
+      source_root File.expand_path("../templates", __FILE__)
+
+      def update_git_ignore
+        append_to_file ".gitignore" do
+          "# React on Rails\nnpm-debug.log\nnode_modules"
```

Comment:
@aaronvb the JS files are generated in production using this: https://github.com/shakacode/react-webpack-rails-tutorial/blob/master/lib/tasks/assets.rake

I think we should copy brakeman, ci, and linters as well from https://github.com/shakacode/react-webpack-rails-tutorial/tree/master/lib/tasks